dashboard/test/dsl/level_dsl_test.rb

```diff
@@ -179,4 +179,31 @@ def setup
     assert_equal 'visible to teachers only', new_level.properties['teacher_markdown']
     assert new_level.encrypted
   end
+
+  test 'editor_experiment set on new markdown level' do
+    old_dsl_text = <<EOS
+name 'new_partner_markdown'
+title 'title'
+description 'description here'
+EOS
+
+    expected_new_dsl_text = <<EOS
+name 'new_partner_markdown'
+editor_experiment 'platformization-partners'
+title 'title'
+description 'description here'
+EOS
```

Comment:
One small style thing: Our ruby version supports [squiggly heredoc](https://infinum.com/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc) now, if the indentation inconsistency bothers you.

Reply:
Thanks for the reminder, @islemaster . I keep forgetting about this and copying the old pattern, so I've changed it in all the DSL-related places I could find.